.github/workflows/ci-reproducibility.yml

```diff
@@ -0,0 +1,38 @@
+# NOTE: This name appears in GitHub's Checks API and in workflow's status badge.
+name: ci-reproducibility
+
+# Trigger the workflow when:
+on:
+  push:
+    branches:
+      - main
+      - stable/*
+  # Besides pushes on the branches above, also check every day at 00:00 UTC.
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+
+  check-reproducible-build:
+    # NOTE: This name appears in GitHub's Checks API.
+    name: check-reproducibility
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Build test binaries, 1.
+        id: build-1
+        uses: ./.github/actions/hash-rust
+        with:
+          binaries: test-runtime-simple-consensus,test-runtime-simple-keyvalue
```

Comment:
This uses the default  `oasisprotocol/runtime-builder:v0.0` image right?
 
Should instead this use the image `oasisprotocol/runtime-builder:<determine_tag_name.output>`instead? So that on `main` it will use the `main` image and on `stable/*` branches the `stable/*` images build by `runtime-builder-image` action.